Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/recovery/core/storage/StoredRecoveryCertificateData.kt

```diff
@@ -7,7 +7,18 @@ data class StoredRecoveryCertificateData(
     @SerializedName("identifier") override val identifier: RecoveryCertificateIdentifier,
     @SerializedName("registeredAt") override val registeredAt: Instant,
     @SerializedName("recoveryCertificateQrCode") override val recoveryCertificateQrCode: String?,
-) : StoredRecoveryCertificate
+) : StoredRecoveryCertificate {
+    override fun equals(other: Any?): Boolean {
+        if (this === other) return true
+        if (javaClass != other?.javaClass) return false
+
+        other as StoredRecoveryCertificateData
+
+        if (identifier != other.identifier) return false
+
+        return true
+    }
+}
```

Comment:
Why the custom `equals` here?

Reply:
because there was no container id yet :)